synapse/tests/test_cortex.py

```diff
@@ -6064,12 +6064,19 @@ async def test_norms(self):
 
     async def test_addview(self):
         async with self.getTestCore() as core:
+            allrole = core.auth.allrole
```

Comment:
checking the role directly works; it is a bit unusual to be testing hiverole directly.
would a lowuser [ without explicit privileges ] being checked if they can read the view work too?

Reply:
yep, that should work. I'll update.